docs/source/integrations/working_with_sql.md

```diff
@@ -0,0 +1,190 @@
+
+# Working with SQL in Kedro
+
+This page outlines the various approaches and recommended practices when building a Kedro pipeline with SQL databases.
+
+## Key concepts and assumptions
+
+When working with SQL databases in Kedro, it's helpful to keep the following concepts front of mind.
+
+### 1. Storage vs. Execution
+
+- You can treat a SQL database purely as a data sink/source, or
+- Leverage the database engine’s compute power whenever possible.
+
+### 2. CRUD Scope
+
+- Kedro handles the parts of CRUD which do not require some concept of “state.”
```

Comment:
Probably worth spelling out CRUD, since it might be a term that not all Kedro users are familiar with